luigi/mock.py

```diff
@@ -170,6 +171,22 @@ def skip(func):
     """
     Sort of a substitute for unittest.skip*, which is 2.7+.
     """
+    if inspect.isclass(func):
+        return object
+
     def wrapper():
         pass
+
     return wrapper
+
+
+def skipIf(condition):
+    """
+
+    """
+    if condition:
+        return skip
+    return lambda x: x
+
+
+skipIfPy3k = skipIf(six.PY3)
```

Comment:
`skip` decorator was already there. It makes to keep those 2 together, no? Maybe `skip`shouldn't be there at the first place. You prefer that I move those `skip`, `skipIf` and `skipIfPy3k` somewhere in /test